src/test/scala/org/hammerlab/guacamole/main/SomaticJointCallerIntegrationTests.scala

```diff
@@ -95,7 +95,7 @@ object SomaticJointCallerIntegrationTests extends SparkCommand[Arguments] with V
         args.out = resultFile
         args.paths = Seq(NA12878TestUtil.subsetBam).toArray
         args.loci = "chr1:0-6700000"
-        args.forceCallLociFromFile = NA12878TestUtil.expectedCallsVCF
+        args.forceCallLociFile = NA12878TestUtil.expectedCallsVCF
```

Comment:
Yea - shouldn't have something failed here? Or because there is no assert this errors quietly?